channel/params_test.go

```diff
@@ -34,9 +34,11 @@ func TestParams_Serializer(t *testing.T) {
 	rng := pkgtest.Prng(t)
 	params := make([]io.Serializer, 10)
 	for i := range params {
-		p := test.NewRandomParams(rng)
+		var p *channel.Params
 		if i&1 == 0 {
-			p.App = channel.NoApp()
+			p = test.NewRandomParams(rng, test.WithApp(channel.NoApp()))
```

Comment:
We have `WithoutApp` for this case which also correctly sets `WithAppData`.

Reply:
done